<?xml version="1.0"?>
<launch>
  <arg name="can_hardware_id" default="013777" />
  <arg name="can_circuit_id" default="0" />
  <arg name="can_bit_rate" default="1000000" />


    <node name="myfile_node" pkg="my_package_1" type="myfile_node" output="screen">
        <param name="can_hardware_id" value="$(arg can_hardware_id)" />
        <param name="can_circuit_id" value="$(arg can_circuit_id)" />
        <param name="can_bit_rate" value="$(arg can_bit_rate)" />
    </node>

</launch>